### DISCUSSION TITLE
Using FieldSet TabbedGroups in custom script?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#16879] [Q&A]
PR #14739 adds more complex form rendering options including TabbedGroups.  Is it possible to use these features in a custom script?  I've been trying to figure out the syntax and so far haven't got it to work.

**Comment by zmaster7** [comment]
I was attempting to do some testing today on this also and it appears the new functionality does not apply to custom scripts. You can still utilize fieldsets to group fields together, but nothing beyond that: https://netboxlabs.com/docs/netbox/en/stable/customization/custom-scripts/#fieldsets

On a side note: I'm not sure if I've run into a bug with it or I'm doing something wrong, but when trying to designate a single field in it's own grouping I end up with an "Internal Server Error" when trying to open the script. If I have 2+ fields in said group it works fine.

**Reply by llamafilm** [reply]
I haven't tried making a fieldset with only one field, but I suspect your issue can probably be solved by adding a comma.  See below example of python syntax:

```py
>>> set1 = ('item')
>>> type(set1)
<class 'str'>

>>> set2 = ('item',)
>>> type(set2)
<class 'tuple'>
```

**Reply by zmaster7** [reply]
That did the trick, thanks!
